graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DefaultDgsQueryExecutor.kt

```diff
@@ -217,3 +162,65 @@ class DefaultDgsQueryExecutor(
         fun reloadSchema(): Boolean
     }
 }
+
+class BaseDgsQueryExecutor {
+    companion object {
```

Comment:
It looks like an candidat for object.
```suggestion
object BaseDgsQueryExecutor {
```

Reply:
Done